### PR TITLE
Add Brocade ICX7250-48 and VDX 6740

### PR DESCRIPTION
**Pull request by rollercow**
Hi,

This adds SNMP discovery support for Brocade:
- ICX7250-48
- VDX 6740

Thanks!

### CODE REVIEW

File: wwwroot/inc/snmp.php [hunk 1]
```diff
@@ -2069,6 +2069,34 @@
...
+	'replacement' => 'Fo /0/\\1',
+	'dict_key' => '10-1588',
+	'label' => '\\1',
+	'try_next_proc' => FALSE,
+);
+
+$iftable_processors['brocade-vdx-SFP+'] = array
+(
+	'pattern' => '@^TenGigabitEthernet 0/(\d+)$@',
+	'replacement' => 'Te /0/\\1',
```
Comment (infrastation) [inline wwwroot/inc/snmp.php]: idem

File: wwwroot/inc/dictionary.php [hunk 2]
```diff
@@ -2476,6 +2476,8 @@ function platform_is_ok ()
 	2585 => array ('chapter_id' => 17, 'dict_value' => '[[Cisco%GPASS%3925E | http://www.cisco.com/c/en/us/products/routers/3925e-integrated-services-router-isr/index.html]]'),
 	2586 => array ('chapter_id' => 17, 'dict_value' => '[[Cisco%GPASS%3945 | http://www.cisco.com/c/en/us/products/routers/3945-integrated-services-router-isr/index.html]]'),
 	2587 => array ('chapter_id' => 17, 'dict_value' => '[[Cisco%GPASS%3945E | http://www.cisco.com/c/en/us/products/routers/3945e-integrated-services-router-isr/index.html]]'),
+	2588 => array ('chapter_id' => 12, 'dict_value' => '[[Brocade%GPASS%VDX 6740 | http://www.brocade.com/en/products-services/switches/data-center-switches/vdx-6740-switches.html]]'),
+	2589 => array ('chapter_id' => 12, 'dict_value' => '[[Brocade%GPASS%ICX-7250-48 | http://www.brocade.com/en/products-services/switches/campus-network-switches/icx-7250-switch.html]]'),
```
Comment (infrastation) [inline wwwroot/inc/dictionary.php]: Could you renumber after the latest changes to this file?

File: wwwroot/inc/snmp.php [hunk 3]
```diff
@@ -2069,6 +2069,34 @@
 	'try_next_proc' => FALSE,
 );
 
+$iftable_processors['brocade-vdx-QSFP+'] = array
+(
+	'pattern' => '@^FortyGigabitEthernet 0/(\d+)$@',
+	'replacement' => 'Fo /0/\\1',
```
Comment (infrastation) [inline wwwroot/inc/snmp.php]: I mean, could you confirm which of the following variants the CLI accepts as a valid interface name?
* `fo /0/0/1`
* `fo 0/0/1`
* `fo0/0/1`

File: wwwroot/inc/snmp.php (hunk 1 above)
Comment (infrastation) [inline wwwroot/inc/snmp.php]: same here

**Comment by infrastation** [comment]
Please review as commented.

File: wwwroot/inc/snmp.php (hunk 3 above)
Comment (rollercow) [inline wwwroot/inc/snmp.php]: So it accepts fo N/0/1 where N is the rbridge id.
As far as I can tell it doesn't expose that via SNMP.

File: wwwroot/inc/snmp.php (hunk 1 above)
Comment (rollercow) [inline wwwroot/inc/snmp.php]: So it accepts fo N/0/1 where N is the rbridge id.
As far as I can tell it doesn't expose that via SNMP.

File: wwwroot/inc/dictionary.php (hunk 2 above)
Comment (rollercow) [inline wwwroot/inc/dictionary.php]: Done.

**Comment by infrastation** [comment]
Those changes are now in maintenance-0.20.x (rebased), thank you! I have made a small fixup, please check if it works for you.

**Pull request closed without merging**